packages/express-backend/routes/entries.js

```diff
@@ -45,7 +45,7 @@ router.get("/:entry_id", async (req, res) => {
 
 //used to look up all public entries
 router.get("/", async (req, res) => {
-  const is_public = req.query.is_public ? req.query.is_public : "False";
+  // const is_public = req.query.is_public ? req.query.is_public : "False";
```

Comment:
I just commented out this line cause it wasn't being used
I can fully delete it later

We handle this logic through the VIEW table in supabase